app/uk/gov/hmrc/minorentityidentificationfrontend/connectors/CreateBusinessVerificationJourneyConnector.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.minorentityidentificationfrontend.connectors
+
+import play.api.http.Status.{CREATED, FORBIDDEN, NOT_FOUND}
+import play.api.libs.json.{JsObject, Json, Writes}
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpReads, HttpResponse, InternalServerException}
+import uk.gov.hmrc.minorentityidentificationfrontend.config.AppConfig
+import uk.gov.hmrc.minorentityidentificationfrontend.connectors.CreateBusinessVerificationJourneyConnector._
+import uk.gov.hmrc.minorentityidentificationfrontend.controllers.trustControllers.{routes => trustControllersRoutes}
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class CreateBusinessVerificationJourneyConnector @Inject()(http: HttpClient,
+                                                           appConfig: AppConfig
+                                                          )(implicit ec: ExecutionContext) {
+
+  def createBusinessVerificationJourney(journeyId: String,
+                                        sautr: String,
+                                        accessibilityUrl: String
+                                       )(implicit hc: HeaderCarrier): Future[BusinessVerificationJourneyCreationResponse] = {
+
+    val jsonBody: JsObject =
+      Json.obj(
+        "journeyType" -> "BUSINESS_VERIFICATION",
+        "origin" -> "vat",
```

Comment:
I think this needs to be the regime parameter from the journey config rather than "vat"